### PR TITLE
chore(router): rename folder to cmf-router

### PR DESCRIPTION
**Pull request by jsomsanith-tlnd**
**What is the problem this PR is trying to solve?**
`ui` repo will soon contains a `router-bridge` package.  
This PR only aims to change `router` folder to `cmf-router` to make the content clearer. 

**What is the chosen solution to this problem?**
Rename `packages/router` to `packages/cmf-router`

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

**Review by romainseb — APPROVED** [review]

**Comment by romainseb** [comment]
Maybe the UMD-size checker has to be updated also ?

**Comment by jsomsanith-tlnd** [comment]
umd size checker takes `'./packages/*/dist/*.{js,css,json}'` as pattern, that should work

**Comment by jsomsanith-tlnd** [comment]
In fact no, router is not part of of the umd size checker, as it executes `prepublishOnly` to build the UMDs, so lerna only.
I wanted to build the router too in prepublishOnly, but I reverted it, let's not build other packages when we need to release tui.

Need to rethink it, but out of scope of this PR

File: .travis/after_success_lint.sh [hunk 1]
```diff
@@ -13,7 +13,7 @@ if [ "$TRAVIS_PULL_REQUEST" != 'false' ]; then
 	(lerna exec --scope=@talend/react-containers -- npm run lint:es -- -o ../../output/containers.eslint.txt --no-color)
 	(lerna exec --scope=@talend/react-forms -- npm run lint:es -- -o ../../output/forms.eslint.txt --no-color)
 	# not in lerna
-	cd packages/router/ && (yarn run lint:es -- -o ../../output/router.eslint.txt --no-color | awk '{if(NR>2)print}') 2>/dev/null && cd ../../
+	cd packages/router/cmf-router && (yarn run lint:es -- -o ../../output/router.eslint.txt --no-color | awk '{if(NR>2)print}') 2>/dev/null && cd ../../
```
Comment (frassinier) [inline .travis/after_success_lint.sh:16]: path seems wrong

File: .github/actions/sourceclear/entrypoint.sh [hunk 2]
```diff
@@ -5,7 +5,7 @@ echo "Script saved to srcclr.sh"
 chmod a+x srcclr.sh
 echo "Script execution rights added"
 
-packages="cmf,cmf-cqrs,cmf-webpack-plugin,components,containers,datagrid,forms,icons,sagas,stepper,theme,dataviz,router"
+packages="cmf,cmf-cqrs,cmf-webpack-plugin,components,containers,datagrid,forms,icons,sagas,stepper,theme,dataviz,router/cmf-router"
```
Comment (frassinier) [inline .github/actions/sourceclear/entrypoint.sh:8]: path seems wrong

File: .travis/after_success_umd.sh [hunk 3]
```diff
@@ -32,8 +32,8 @@ cp -R packages/icons/dist/* .static/@talend/icons/dist
 echo "✓ Copy icons UMD to .static"
 
 mkdir -p .static/@talend/react-cmf-router/dist
-cp -R packages/router/dist/* .static/@talend/react-cmf-router/dist
-echo "✓ Copy router UMD to .static"
+cp -R packages/router/cmf-router/dist/* .static/@talend/react-cmf-router/dist
```
Comment (frassinier) [inline .travis/after_success_umd.sh:35]: path seems wrong

**Review by frassinier — CHANGES_REQUESTED** [review]

File: .travis/after_success_lint.sh (hunk 1 above)
Comment (jsomsanith-tlnd) [reply .travis/after_success_lint.sh]: Thanks !

**Review by frassinier — APPROVED** [review]

**Pull request merged**